runtime/fail_byt.c

```diff
@@ -67,7 +72,7 @@ CAMLexport void caml_raise_with_args(value tag, int nargs, value args[])
 {
   CAMLparam1 (tag);
   CAMLxparamN (args, nargs);
-  value bucket;
+  CAMLlocal1(bucket);
```

Comment:
Revert to trunk.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/7d2d9e0a2cf1969b1ca0e621bca4cc2534b88d91